internal/pkg/config/v1/config.go

```diff
@@ -154,6 +154,19 @@ func (c *Config) DecryptContextStates() error {
 	return c.Validate()
 }
 
+func (c *Config) DecryptCredentials() error {
+	if credentials := c.Credentials; c.Credentials != nil {
+		for _, credential := range credentials {
+			if apiKeyPair := credential.APIKeyPair; apiKeyPair != nil {
+				if err := apiKeyPair.DecryptCredentialAPISecret(); err != nil {
+					return err
+				}
+			}
```

Comment:
No reason to create another variable, IMO
```suggestion
			if credential.APIKeyPair != nil {
				if err := credentialAPIKeyPair.DecryptCredentialAPISecret(); err != nil {
					return err
				}
			}
```